labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,7 +33,40 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    lines.map((A) => A.map((B) => cypher(B)))
+  }
+  //I can't find a pattern here. so....
+  def cypher(letter:Char): Char = {
+    letter match {
+      case 'y' => 'a'
+      case 'n' => 'b'
+      case 'f' => 'c'
+      case 'i' => 'd'
+      case 'c' => 'e'
+      case 'w' => 'f'
+      case 'l' => 'g'
+      case 'b' => 'h'
+      case 'k' => 'i'
+      case 'u' => 'j'
+      case 'o' => 'k'
+      case 'm' => 'l'
+      case 'x' => 'm'
+      case 's' => 'n'
+      case 'e' => 'o'
+      case 'v' => 'p'
+      case 'z' => 'q'
+      case 'p' => 'r'
+      case 'd' => 's'
+      case 'r' => 't'
+      case 'j' => 'u'
+      case 'g' => 'v'
+      case 't' => 'w'
+      case 'h' => 'x'
+      case 'a' => 'y'
+      case 'q' => 'z'
+      case ' ' => ' '
+        
+    }
```

Comment:
Haha I really liked this approach to the question, although I don't think it's very Scala-y. I think the same could have been done if you created copied and pasted the input and output strings provided and created a map from that, something like input zip output toMap. The toMap feature is super nice and just takes a pair and automatically makes it key-value pairs in a map. It was definitely less verbose than the provided solution. Also technically this is pattern matching, so maybe it is still Scala-y?